### PR TITLE
Added initial code of popup-task for rewiew

### PR DESCRIPTION
**Pull request by Iakow**
### HTML & CSS practice: Hooli-style Popup

[Demo](https://iakow.github.io/hooli-popup/) |
[Code base](https://github.com/Iakow/frontend-2021-homeworks/tree/html-css-popup/submissions/iakow/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/iakow/html-css-popup/style.css [hunk 1]
```diff
@@ -0,0 +1,255 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
```
Comment (MaxKalashnyk) [inline submissions/iakow/html-css-popup/style.css:5]: It's better to apply `list-style: none;` for the ul element

File: submissions/iakow/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  margin: 10px;
+  border: none;
+  background-color: #ffffff40;
+}
+
+.hidden, .popup-bg-closer {
+  display: none;
+}
+
+.hidden, .popup-bg-closer {
```
Comment (MaxKalashnyk) [inline submissions/iakow/html-css-popup/style.css:127]: Remove these styles, please

File: submissions/iakow/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      </li>
+
+      <li class="relative-box" title="Applications">
+        <input class="popup-ctrl" id="popup-ctrl" type="checkbox">
+
+        <label class="nav-item apps" for="popup-ctrl"></label>
+        <label class="popup-bg-closer" for="popup-ctrl"></label>
+
+        <div class="popup">
+          <a class="popup-item maps" href="#">Maps</a>
```
Comment (MaxKalashnyk) [inline submissions/iakow/html-css-popup/index.html:29]: I'd recommend creating some span element inside the `a` tag and set a background image for `span`

File: submissions/iakow/html-css-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      </li>
+
+      <li title="Images">
+        <a class="nav-item images" href="#"></a>
+      </li>
+
+      <li class="relative-box" title="Applications">
+        <input class="popup-ctrl" id="popup-ctrl" type="checkbox">
+
+        <label class="nav-item apps" for="popup-ctrl"></label>
```
Comment (MaxKalashnyk) [inline submissions/iakow/html-css-popup/index.html:25]: Why do you need two labels here?

File: submissions/iakow/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+  background-repeat: no-repeat;
+  background-position: center center;
+  cursor: pointer;
+}
+
+.nav-item:hover {
+  background-color: rgb(98 98 98);
+}
+
+.popup-ctrl, .popup-expansion {
```
Comment (MaxKalashnyk) [inline submissions/iakow/html-css-popup/style.css:45]: It's better to use these styles for hiding `input`:
``` CSS
.visually-hidden {
	position: absolute !important;
	clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
	clip: rect(1px, 1px, 1px, 1px);
	padding:0 !important;
	border:0 !important;
	height: 1px !important; 
	width: 1px !important; 
	overflow: hidden;
}
```
More info [here](https://htmlacademy.ru/blog/boost/frontend/short-12)

**Review by MaxKalashnyk — CHANGES_REQUESTED** [review]
Good job! Special respect for Silicon Valley characters! 
See my comments below, please

File: submissions/iakow/html-css-popup/index.html (hunk 4 above)
Comment (Iakow) [reply submissions/iakow/html-css-popup/index.html:25]: To be able to close the popup by clicking anywhere else in the document.

Comment (MaxKalashnyk) [reply submissions/iakow/html-css-popup/index.html:25]: Got it! Ok!

File: submissions/iakow/html-css-popup/style.css (hunk 1 above)
Comment (Iakow) [reply submissions/iakow/html-css-popup/style.css:8]: Done!

File: submissions/iakow/html-css-popup/style.css (hunk 2 above)
Comment (Iakow) [reply submissions/iakow/html-css-popup/style.css]: Done!

File: submissions/iakow/html-css-popup/index.html (hunk 3 above)
Comment (Iakow) [reply submissions/iakow/html-css-popup/index.html]: Seems to be done )

File: submissions/iakow/html-css-popup/style.css (hunk 5 above)
Comment (Iakow) [reply submissions/iakow/html-css-popup/style.css:48]: Thanks! Done.

**Comment by Iakow** [comment]
Should I press the resolv buttons?

**Comment by MaxKalashnyk** [comment]
> Should I press the resolv buttons?

Yes!

**Review by MaxKalashnyk — APPROVED** [review]
LGTM! Nice work!

**Comment by Iakow** [comment]
Thank you! GL )

**Pull request merged**